packages/stg-evm-v2/devtools/config/mainnet/01/usdt-token.config.ts

```diff
@@ -0,0 +1,46 @@
+import assert from 'assert'
+
+import { TokenName } from '@stargatefinance/stg-definitions-v2'
+import { USDCNodeConfig } from '@stargatefinance/stg-devtools-v2'
+
+import { OmniGraphHardhat, createContractFactory, createGetHreByEid } from '@layerzerolabs/devtools-evm-hardhat'
+import { EndpointId } from '@layerzerolabs/lz-definitions'
+
+import { createGetAssetAddresses, getAssetNetworkConfig } from '../../../../ts-src/utils/util'
+import { onPeaq } from '../utils'
+
+const fiatContract = { contractName: 'FiatTokenV2_2' }
+
+// For external USDT deployments
+const usdtPeaqAsset = getAssetNetworkConfig(EndpointId.PEAQ_V2_MAINNET, TokenName.USDT)
+assert(usdtPeaqAsset.address != null, `External USDT address not found for PEAQ`)
+
+// TODO change USDCNodeConfig to USDTNodeConfig, but that does not yet exist
+export default async (): Promise<OmniGraphHardhat<USDCNodeConfig, unknown>> => {
```

Comment:
Is it worth adding a `USDTNodeConfig` if this script works? I think we should but unsure of how big of a lift it is. Will look into this

Reply:
I think there should be an `OwnableConfig` type already for this in `ua-devtools`